packages/runtime/src/types/transport/RecipientDTO.ts

```diff
@@ -2,5 +2,5 @@ export interface RecipientDTO {
     address: string;
     receivedAt?: string;
     receivedByDevice?: string;
-    relationshipId: string;
+    relationshipId?: string;
```

Comment:
Why is this optional now? Would you mind giving me a little context?

Reply:
I think this is caused by the anonymized messages. We don't have relationships for such recipients.

Reply:
Ah, right. Cheers 👍